packages/dashboard/src/components/widgets/tile/contants.ts

```diff
@@ -0,0 +1 @@
+export const USE_CASE_DELAY_IN_SECS = 3000;
```

Comment:
Constant name is confusing. How about something like DATA_DELAY_AFTER _REQUEST_IN_SECS